src/components/main/TodoItem.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const TodoItem = ({ todos, changeStatus, deleteTodo, filter }) => (
+  <div>
+    {todos
+      .filter((todo) => {
+        if (filter === 'active') {
+          return (todo.status === false);
+        }
+
+        if (filter === 'completed') {
+          return (todo.status === true);
+        }
+
+        return (todo.status === false || todo.status === true);
```

Comment:
```suggestion
        return todo;
```
Что за проверки. если тебе любой todo подойдет?